### PR TITLE
Add method objectState.RemoteOrLocalState

### PR DESCRIPTION
**Pull request by webrouse**
Part of: https://keboola.atlassian.net/browse/COM-1104

### CODE REVIEW

File: internal/pkg/model/state.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ type ObjectState interface {
 	SetRemoteState(object Object)
 	RemoteState() Object
 	LocalOrRemoteState() Object
+	RemoteOrLocalState() Object
```
Comment (webrouse) [inline internal/pkg/model/state.go:33]: Pridana metoda, ktora vrati `Remote` stav, ... ak nie je znamy tak `Local` stav.

File: internal/pkg/model/state.go [hunk 2]
```diff
@@ -200,6 +201,39 @@ func (r *ConfigRowState) LocalOrRemoteState() Object {
...
+	case b.HasLocalState():
+		return b.LocalState()
+	default:
+		panic(fmt.Errorf("object Remote or Local state must be set"))
+	}
+}
+
+func (c *ConfigState) RemoteOrLocalState() Object {
+	switch {
+	case c.HasLocalState():
```
Comment (kacurez) [inline internal/pkg/model/state.go:217]: nema tu byt ako prvy `case c.HasRemoteState`?

Comment (webrouse) [reply internal/pkg/model/state.go:217]: mal, idem fixnut, dik

Comment (webrouse) [reply internal/pkg/model/state.go]: https://github.com/keboola/keboola-as-code/pull/135/commits/f9137dbeb29d1f7c996ac8013d3b8f5fc7e61494

**Review by kacurez — APPROVED** [review]
lgtm

**Pull request merged**